lib/iris/common/resolve.py

```diff
@@ -144,7 +144,7 @@ class Resolve:
                 forecast_reference_time     1859-09-01 06:00:00
                 height                      1.5 m
             Cell methods:
-                mean                        time (6 hour)
+                0                           time: mean: (interval: 6 hour)
```

Comment:
I reckon that the extra colon here is not correct.
It should be just `"time: mean (interval: 6 hour)"`

Reply:
Oh, I think this was maybe already pointed out in [this comment](https://github.com/SciTools/iris/pull/5224/files#r1154437823)